src/gui/mzroll/alignmentdialog.cpp

```diff
@@ -93,6 +103,13 @@ void AlignmentDialog::intialSetup() {
 	algoChanged();
 	minIntensity->setValue(_mw->mavenParameters->minIntensity);
 	maxIntensity->setValue(_mw->mavenParameters->maxIntensity);
+
+    samplesBox->clear();
+    samplesBox->addItem("Select Randomly",QVariant(QVariant::fromValue(static_cast<void*>(nullptr))));
+    for(auto sample: _mw->samples) {
+        if(sample->isSelected)
+            samplesBox->addItem(sample->sampleName.c_str(), QVariant(QVariant::fromValue(static_cast<void*>(sample))));
```

Comment:
Can we do a sort-by-name on these samples once before adding them to the drop-down list? On my system the samples are appearing in the list in no particular order.

Reply:
I agree that the samples should be displayed in some order but I don't think this is the right place to sort them. The order of samples is messed up in other parts as well. 
Ideally `_mw->samples` should be stored in sorted order or in the way user orders them. This ordering should be displayed in the `ProjectDockWidget` and others widget should be synced with it.